lib/matplotlib/backends/backend_agg.py

```diff
@@ -22,8 +22,6 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
-import six
```

Comment:
Please leave six everywhere.  Current policy is every file gets the future import + six.